mythguy1226/80sgame
# fixed accuracy bug

Changes:
* Accuracy in Classic mode now should not exceed 100
* Activated Power Cores no longer count towards shots/accuracy
* Click in pause menu no longer count as shots
* Accuracy of the final round is now considered

To test:
* Open classic mode and play through
* Keep a rough estimate of your accuracy as you play
* Make sure the report and your track roughly match
* If necessary, lower round count to test

Issue: https://bat-bots.atlassian.net/jira/software/projects/BB/boards/1?selectedIssue=BB-241

Due Sprint 5
